templates/server/src/main/resources/archetype-resources/__earProjectName__/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>${groupId}</groupId>
 		<artifactId>${artifactId}</artifactId>
-		<version>${version}</version>
+	    <version>${app.version}</version>
```

Comment:
is our devonfw convention with 2 spaces indent so hard to handle?
I will fix it on github and merge the PR to avoid wasting time...